analyzers/tests/SonarAnalyzer.UnitTest/Extensions/SafeCSharpSyntaxWalkerTest.cs

```diff
@@ -29,6 +29,8 @@ public class SafeCSharpSyntaxWalkerTest
         public void GivenSyntaxNodeWithReasonableDepth_SafeVisit_ReturnsTrue() =>
             new Walker().SafeVisit(SyntaxFactory.ParseSyntaxTree("void Method() { }").GetRoot()).Should().BeTrue();
 
+#if NET
+
         [TestMethod]
         public void GivenSyntaxNodeWithHighDepth_SafeVisit_ReturnsFalse()
```

Comment:
This test is now failing on .Net Framework. Could be because of the instrumentation changes but I cannot really tell.

Reply:
If they randomly fail we should add a github issue. In my case they were consistently failing when running in AltCover context,

Reply:
They should anyway behave the same for all target frameworks if I understand correctly. Will you add an issue for investigating them?

Reply:
As far as I understand we are talking about 2 separate problems:
- random fails on the local machine (happened for you) -> I think it would be best that you log this
- consistent fails when running in AltCover context with net framework 4.8. I think it's not worth investigating this since the case is tested on dotnet core.

Reply:
what error did this test fail with, could you add it here?

Reply:
The error was `Expected boolean to be false, but found True.` So, not a very useful message.

Reply:
> As far as I understand we are talking about 2 separate problems:
> 
> * random fails on the local machine (happened for you) -> I think it would be best that you log this
> * consistent fails when running in AltCover context with net framework 4.8. I think it's not worth investigating this since the case is tested on dotnet core.

I have opened an issue for it: https://github.com/SonarSource/sonar-dotnet/issues/6174

Reply:
`Expected boolean to be false, but found True.`